libs/api/domains/hms-loans/src/lib/models/paymentOverview.model.ts

```diff
@@ -0,0 +1,34 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType('HmsLoansPaymentOverview')
+export class PaymentOverview {
```

Comment:
Same here for payments.
```suggestion
@ObjectType('HmsLoansPayment')
export class Payment {
```

Reply:
Fixed